src/modules/workspace/promote.ts

```diff
@@ -29,21 +29,21 @@ const isPromotable = async (workspace: string) => {
     )
   }
 }
-const promptPromoteConfirm = (workspace: string): Promise<any> =>
+const promptPromoteConfirm = (workspace: string): Promise<boolean> =>
   promptConfirm(`Are you sure you want to promote workspace ${chalk.green(workspace)} to master?`, true).then(
     answer => {
-      if (!answer) {
-        throw new UserCancelledError()
-      }
+      return answer
     }
   )
 
 export default async () => {
   log.debug('Promoting workspace', currentWorkspace)
   await isPromotable(currentWorkspace)
-  await promptPromoteConfirm(currentWorkspace)
-  await promote(account, currentWorkspace)
+  const promptAnswer = await promptPromoteConfirm(currentWorkspace)
+  if (promptAnswer) {
+    await promote(account, currentWorkspace)
 
-  log.info(`Workspace ${chalk.green(currentWorkspace)} promoted successfully`)
-  await useCmd('master')
+    log.info(`Workspace ${chalk.green(currentWorkspace)} promoted successfully`)
+    await useCmd('master')
```

Comment:
_super nit and maybe unnecessary just if you want_: The vtex-eslint likes to do early returns in some cases, like this:
```
const promptAnswer = ...
if (!promptAnswer) {
  return
}

// do something
```
But I think it's okay the way it is

Reply:
I tried that, but lint doesn't allow blocks with only `return`. We could disable this lint rule, though. What do you think?

Reply:
For me it's ok to disable!